torch/quantization/observer.py

```diff
@@ -238,10 +312,48 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
         super(MinMaxObserver, self)._load_from_state_dict(state_dict, prefix, local_metadata, False,
                                                           missing_keys, unexpected_keys, error_msgs)
 
+
 class MovingAverageMinMaxObserver(MinMaxObserver):
-    def __init__(self, averaging_constant=0.01, **kwargs):
+    r"""Observer module for computing the quantization parameters based on the
+    moving average of the min and max values.
+
+    This observer computes the quantization parameters based on the moving
+    averages of minimums and maximums of the incoming tensors. The module
+    records the average minimum and maximum of incoming tensors, and uses this
+    statistic to compute the quantization parameters.
+
+    Args:
+        averaging_constant: Averaging constant for min/max.
+        dtype: Quantized data type
+        qscheme: Quantization scheme to be used
+        reduce_range: Reduces the range of the quantized data type by 1 bit
+
+    The moving average min/max is computed as follows
+
+    .. math::
+
+        x_\text{min/max} = \begin{cases}
+            \frac{\min}{\max}(X) & \text{if~}x_\text{min/max} = \O \\
+            (1 - c) x_\text{min/max} + c \frac{\min}{\max}(X) & \text{otherwise}
+        \end{cases}
```

Comment:
![image](https://user-images.githubusercontent.com/4216323/66863915-e0389880-ef48-11e9-8eb0-4894805786c4.png)

Reply:
Same comments as before for min and max, Also, lets use None instead of phi